ProcessLib/SmallDeformation/SmallDeformationFEM.h

```diff
@@ -39,9 +41,22 @@ struct IntegrationPointData final
     {
     }
 
-#if defined(_MSC_VER) && _MSC_VER < 1900
-    // The default generated move-ctor is correctly generated for other
-    // compilers.
+    explicit IntegrationPointData(IntegrationPointData const& other)
```

Comment:
As suggested [here](https://eigen.tuxfamily.org/dox/group__TopicStructHavingEigenMembers.html#othersolutions), IMHO a private structure containing only Eigen members would be a solution superior to the `clone()` method.
But maybe even removing the `<Eigen/StdVector>` include makes `clone()` unnecessary.